service/src/main/java/com/fecklessweasel/service/FileUploadServlet.java

```diff
@@ -0,0 +1,85 @@
+package com.fecklessweasel.service;
+
+import com.fecklessweasel.service.datatier.SQLInteractionInterface;
+import com.fecklessweasel.service.datatier.SQLSource;
+import com.fecklessweasel.service.objectmodel.FileMetadata;
+import com.fecklessweasel.service.objectmodel.ServiceException;
+import com.fecklessweasel.service.objectmodel.UserSession;
+
+import java.io.*;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.Date;
+import javax.servlet.annotation.MultipartConfig;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.Part;
+
+@WebServlet("/servlet/file/upload")
+@MultipartConfig
+public class FileUploadServlet extends HttpServlet {
+
+    private static final String FILEPATH_PREFIX = "/files";
+
+    @Override
+    protected void doPost(final HttpServletRequest request,
+                          final HttpServletResponse response)
+            throws ServletException, IOException {
+
+        // TODO: These should go into database eventually
+        String title = request.getParameter("title");
```

Comment:
Do you want to go ahead and make this change? If you are unsure either I or @hschmack can help you with it. If not, open a work item.